doc-content/enterprise-only/employee-rostering/er-rotation-edit-proc.adoc

```diff
@@ -1,14 +1,18 @@
 [id='er-rotation-edit-proc']
 = Viewing and editing the rotation pattern
 
-The rotation pattern contains shifts that are automatically replicated for every period. For details about the rotation pattern, see <<er-rotation-con>>.
+The rotation pattern contains shift templates that are automatically replicated for every period. For details about the rotation pattern, see <<er-rotation-con>>.
 
 You can view the rotation pattern, add, move, and delete shifts.
 
 .Procedure
```

Comment:
This is an example where I'd try to reduce so many "To do X" instances and instead walk the user through the overall task. Some instances are fine, but when every step is "To do X", then it's really not a procedure but a list of options, and the module is more of a ref than a proc.

Here's my whack at it. I realize in a couple of cases I just reversed the sentence structure, but it walks the user immediately through what to do rather than give so many options.

```
. Click the *Rotation* tab to view and edit the rotation pattern.
. Use the horizontal scroll bar at the bottom of the browser window to view additional days of the rotation pattern.
. Click an empty area of the table and add a shift template. The application adds a shift, determining the slot and time span automatically from the location of the click.
. To change the start or end time of a shift template, move the mouse pointer to the left or right border of the box representing the shift template and drag the border.
. Click the shift template and edit the following values as needed:
** *From* and *To* day and time: The exact time and duration of the shift. The day number is counted from the start of the rotation period.
** *Rotation employee*: The employee assigned to the shift by default. Select *Unassigned* if you do not want to assign any employee to this shift by default; in this case, the employee in the shift is determined by the rostering process in every case without a preferred value.
+
. Click *Apply* to save the changes.
. To delete a shift template, hold the *Shift* key on the keyboard and click the shift template. Alternatively, click the shift template and then click *Delete shift template*.
```

IBM talks about this but I won't take the time to hunt it down, given the time constraints. Feel free to disregard this for now if you'd like.

Reply:
The procedure as written follows creating one shift template and would need a "rinse and repeat"; I see no obvious way to add this "rinse and repeat". Leaving until BXMSDOC-3906.

Reply:
@mramendi you can always start with the disclaimer like "To create a shift template"..." That should tell the users that they need to repeat this to create more than  one. It also gives them the benefit of doubt. I like what @sterobin suggests here.
Since we are moving towards FCC, approaching topics with this intent is always good.